health_check/views.py

```diff
@@ -7,15 +7,15 @@
 
 def home(request):
```

Comment:
How about we name this view to `index` and give name it `healthcheck:index` so we won't have multiple different terms about it floating around and have correct URL namespacing for it as well. This way we won't clash with any view names in other applications.